### DISCUSSION TITLE
Broadcast message to all online players

### DISCUSSION THREAD

**Discussion by malafk** [Minestom/Minestom#1333] [Help] [answered]
Not sure how to do this the proper way, I know you can do a loop but I remember theres a way with audiences.. thanks.

**Comment by emortaldev** [comment, marked as answer]
Audiences.all().sendMessage(<component>)

**Comment by KrystilizeNevaDies** [comment]
`Audiences` contains a lot of useful collections for this. You get the audience object and then send the message to it.
